testsuite/openmodelica/cppruntime/fmu/modelExchange/2.0/testCSTR.mos

```diff
@@ -66,6 +66,44 @@ getErrorString();
 //   <ModelExchange
 //     modelIdentifier=\"CSTRModel\" providesDirectionalDerivative=\"true\">
 //   </ModelExchange>
+//   <UnitDefinitions>
+//     <Unit name =\"s\">
+//       <BaseUnit  s=\"1\"/>
+//     </Unit>
+//     <Unit name =\"kg/m3\">
+//       <BaseUnit  m=\"-3\" kg=\"1\" factor=\"1000.0\"/>
```

Comment:
I guess the factor is wrong here. The unit checking module is probably working with gram instead of kilogram.

Reply:
So we should display it as grams instead of kg ?

Reply:
No, we should display it as kilogram:
```xml
<BaseUnit  m=\"-3\" kg=\"1\" factor=\"1.0\"/>
```

Reply:
So if the base units contains some value for kg , then i should divide the factor by 1000 or display the default factor value which is "1", is this correct

Reply:
We have to multiply the factor with `10^(-3*kg)` and kg stands for the exponent from the base unit. (The exponent is 1 in all the tests, so we should also test other exponents.) However, we could also just change it directly in the unit module instead.

Reply:
This is fixed now